account-management-app/src/main/java/org/duracloud/account/app/controller/UsersController.java

```diff
@@ -152,6 +161,73 @@ public ModelAndView changeUserRole(@PathVariable("id") Long userId,
         return createRedirectMav(BASE_VIEW);
     }
 
+    @RequestMapping(value = CREATE_LIST_VIEW, method = RequestMethod.GET)
+    public String createList(HttpServletResponse response) throws Exception {
+        log.info("Creating DuraCloud users list.");
+        String csvHeaders = "First Name,Last Name,Username,Email,Accounts & Roles\n";
+
+        StringBuffer contentDisposition = new StringBuffer();
+        contentDisposition.append("attachment;");
+        SimpleDateFormat dateFormat =
+            new SimpleDateFormat(DateUtil.DateFormat.PLAIN_FORMAT.getPattern());
+        contentDisposition.append("filename=\"duracloud-users-" + dateFormat.format(new Date()) + ".csv\"");
+        response.setHeader("Content-Disposition", contentDisposition.toString());
+
+        List<String[]> users = this.buildUsersList();
+        log.info("Found {} users.", users.size());
+
+        try {
+            response.getOutputStream().print(csvHeaders);
+            ListIterator<String[]> itr = users.listIterator();
+            while (itr.hasNext()) {
+                String user = StringUtils.join(itr.next(), ",");
+                response.getOutputStream().print(user + "\n");
+            }
+        } catch (IOException ex) {
+            ex.printStackTrace();
```

Comment:
It would be better to use `log.error()` here, so we make sure the error is captured properly in the log.